docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
       - DOCKERIZED=true
     env_file:
       - '.env'
+    tty: true
```

Comment:
I assume this was intentional but wanted to double check this didnt just happen. Seems like it's to help pry hang?